metricbeat/module/uwsgi/_meta/config.yml

```diff
@@ -1,4 +1 @@
 - module: uwsgi
-  metricsets: ["status"]
-  period: 10s
-  hosts: ["tcp://127.0.0.1:9191"]
```

Comment:
did you remove the host intentionally? Our `config.yml` gets shorter and short. There could be arguments to also remove the hosts part. @andrewkroh ?

Reply:
I was thinking that this was the default, I was wrong :) I'll readd the `host` entry.

Reply:
Well, in the documentation it says this is the default, but I cannot find it in the code, I'm going to take a look to this so doc and code are aligned.

Reply:
I have readded the `host` entry, and rephrashed doc to mention `tcp://127.0.0.1:9191` as example instead of default.